### PR TITLE
Change configuration access method from Get() to global variable 'Config'

### PR DESCRIPTION
**Pull request by kuramal**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test

 /kind feature

**What this PR does / why we need it**:

Change configuration access method from Get() to global variable 'Config'

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cloud/pkg/edgecontroller/manager/configmap.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package manager
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (kevin-wangzefeng) [inline cloud/pkg/edgecontroller/manager/configmap.go]: No usage referring to this change?

Comment (kuramal) [reply cloud/pkg/edgecontroller/manager/configmap.go]: Yeah, it is not a necessary modification. It has been restored

**Pull request merged**